dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/utils/ProcessUtils.java

```diff
@@ -146,6 +147,32 @@ public static String getPidsStr(int processId) throws Exception {
         return sb.toString().trim();
     }
 
+    public static String parsePidStr(String rawPidStr) {
```

Comment:
This function has some common logic with function `getPidsStr`. I think the common logic should be extracted.